bdai_ros2_wrappers/bdai_ros2_wrappers/action.py

```diff
@@ -21,7 +21,9 @@
 class ActionException(Exception):
     """Base action exception."""
 
-    pass
+    def __init__(self, action: "ActionFuture") -> None:
+        super().__init__(action)
+        self.action = action
 
 
 class ActionTimeout(ActionException):
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 41 to 41]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Should we provide a string elaborating on the error, like other exceptions do? <!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/bdaiinstitute/ros_utilities/63?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
I guess we could. I didn't because the exception types themselves are pretty self explanatory, and the action future has almost full access to action details.

Reply:
I'm ok with not. There isn't a generic way to pull the status msg out of the action result and since we are providing that, the use can do it themselves